packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/TreeHierarchy.tsx

```diff
@@ -91,6 +91,9 @@ export const TreeHierarchy = ({
   const isLoadingOlderMessagesRef = useRef(false);
   const previousCommentsLengthRef = useRef(0);
   const [isLoadingOlderMessages, setIsLoadingOlderMessages] = useState(false);
+  const [loadButtonClicked, setLoadButtonClicked] = useState(false);
```

Comment:
problem with this solution is that when you reply to comment that already is nested, the reply won't be shown right away, which is bad UX

https://github.com/user-attachments/assets/0dedc353-7476-4a87-aa97-8ba094393016